apps/opik-backend/src/main/java/com/comet/opik/domain/ProjectService.java

```diff
@@ -271,17 +285,36 @@ public Page<Project> find(int page, int size, @NonNull ProjectCriteria criteria,
             return traceDAO.getLastUpdatedTraceAt(projectIds, workspaceId, connection);
         }).block();
 
+        List<UUID> projectIds = projectRecordSet.content.stream().map(Project::id).toList();
+
+        Map<UUID, Map<String, Object>> projectStats = getProjectStats(projectIds, workspaceId);
+
         List<Project> projects = projectRecordSet.content()
                 .stream()
-                .map(project -> project.toBuilder()
-                        .lastUpdatedTraceAt(projectLastUpdatedTraceAtMap.get(project.id()))
-                        .build())
+                .map(project -> enhanceProject(project, projectLastUpdatedTraceAtMap, projectStats))
```

Comment:
Due to the in-memory sorting for the `lastUpdatedTraceAt`, there's another code path that should be added with the `enhanceProject()` logic. See line 264 above (can't leave my comment there since there is no change)

Reply:
Done in https://github.com/comet-ml/opik/pull/894/commits/aa8c35f64aac2a3e4c06488ef7ee4998aa215cde, covered in https://github.com/comet-ml/opik/pull/894/commits/8563ae6e4a348f5a14b8b0d100f94accac65c864